### ISSUE TITLE
Tooltips are not visible for columns on small canvas with shadows enabled

### ISSUE THREAD

**Issue by dmitrygorbenko** [highcharts/highcharts#17357]
#### Live demo with steps to reproduce
1. Open https://jsfiddle.net/nbu3g8mj/
2. Try to hover columns
3. If you still see the tooltip, then maybe you are lucky to have a wide screen, so add "width: 500, height: 500" to "chart" section of the config object.

#### Expected behaviour
When you mouse hover columns the tooltip appears.

#### Actual behaviour
Tooltips do not appear when I mouse hover columns.

#### Product version
10.1.0

#### Affected browser(s)
Chrome

#### Notes
I've noticed that the toltips may appear if I make my screen wider, so the columns get also wider, and shadows do not catch mouse events. If you don't have wider screen, then just try to disable some series by clicking on them in the Legend area - by disabling some of them (like, leaving 5 active series on canvas) eventually you'll be able to hover columns and see tooltips over them.

But, it does not depend on column's width only, because if you open this sample https://jsfiddle.net/qm926sy8/ you can see much more columns and they display tooltips. So, something has to be related to values in the series (in my sample I have nulls).

**Comment by raf18seb** [comment]
Hi @dmitrygorbenko, thanks for reporting the issue!
ps. Just to clarify - it's SVG, not canvas :) It's relevant here, because the SVG elements are added to the DOM in the wrong order.

Simplified demo: https://jsfiddle.net/BlackLabel/c815uvqe/

It seems that it's caused by the `accessibility.js` module.

Without the accessibility module, the point with shadows are added to DOM in this order:
![image](https://user-images.githubusercontent.com/23310742/172141525-e56fa465-da77-45b4-9573-afe4138e9844.png)

With the accessibility module added, the order is reversed, and the shadow with `stroke-width: 5` covers the point:
![image](https://user-images.githubusercontent.com/23310742/172142011-36b5d95e-bd09-421d-aab3-ff9b6ac15e00.png)

The simplest and most obvious workaround is turning off the accessibility feature.

**Comment by oysteinmoseng** [comment]
Internal note: Probably happens because of this:
https://github.com/highcharts/highcharts/blob/master/ts/Accessibility/Components/SeriesComponent/SeriesDescriber.ts#L700-L706
